tests/mountingOptions/stubs.global.spec.ts

```diff
@@ -607,7 +607,7 @@ describe('mounting options: stubs', () => {
       )
     })
 
-    it('renders the default slot of deeply nested stubs when renderStubDefaultSlot=true', () => {
+    it('renders the default slot of deeply nested stubs when renderStubDefaultSlot=true', async () => {
```

Comment:
Not needed.

Reply:
oh...my gosh... Sorry for my fault. I'll remove this.

Reply:
You apologized twice - once in your PR description and once here. No need to do that! You are doing the world a favor with your PR, if anything everyone should be thanking *you*. 🎉 Thanks a lot!